### PR TITLE
Add _toc.yml and _config.yml

### PR DESCRIPTION
**Pull request by aakankshaduggal**
This Pull Request addresses #53

### CODE REVIEW

File: _toc.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+root: README
+parts:
+  - caption: Getting started
+    chapters:
+    - file: docs/get-started
+    - file: docs/how-to-contribute
+  - caption: AIOps projects
+    chapters:
+    - file: docs/aiops-projects
+    - file: docs/project-document-template
```
Comment (MichaelClifford) [inline _toc.yml:11]: Can we re-title this here or change the header name on the doc to "Project outline template"?

File: _toc.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    - file: docs/project-structure
```
Comment (MichaelClifford) [inline _toc.yml:12]: Same here, update the title to remove the "-" character at the end

File: _toc.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  - caption: Best Practices and Guidelines
+    chapters:
+    - file: docs/guidelines-for-reviewing-datascience-code.md
+    - file: docs/data-visualization-best-practices.md
```
Comment (MichaelClifford) [inline _toc.yml:16]: Also here updated it so that this does not have bold text on the jupyterbook sidebar

**Review by MichaelClifford — CHANGES_REQUESTED** [review]

File: _toc.yml (hunk 1 above)
Comment (aakankshaduggal) [reply _toc.yml:11]: Afaik, the title is coming from the docs. Would you suggest changing the names of the documents here?

Comment (MichaelClifford) [reply _toc.yml:11]: yes please : )

Comment (aakankshaduggal) [reply _toc.yml:11]: Awesome, will do that :+1:

**Comment by aakankshaduggal** [comment]
Resolved it @MichaelClifford :+1:

**Review by MichaelClifford — APPROVED** [review]

**Comment by MichaelClifford** [comment]
/approve
Thanks!

**Pull request merged**